indico/modules/events/registration/controllers/management/reglists.py

```diff
@@ -164,9 +175,12 @@ class RHRegistrationsActionBase(RHManageRegFormBase):
 
     registration_query_options = ()
 
-    def _process_args(self):
+    @use_kwargs({
+        'registration_id': fields.List(fields.Integer(), missing=[]),
```

Comment:
I'd use the name mapping (like in the example I showed you) in order to call the attribute `registration_ids`:

    'registration_ids': fields.List(fields.Int(), data_key='registration_id', missing=[]),